src/app/storymaps/tpl/core/Maptiks.js

```diff
@@ -0,0 +1,21 @@
+define(["dojo/topic"],
+  function(topic){
+    topic.subscribe("story-loaded-map", function(){
+      require(['maptiks'], function (mapWrapper) {
+        if (!app.map.maptiks) {
+          var container = app.map.container; // the current map div
+          var maptiksSet = app.data.getWebAppData().getMaptiks().hasOwnProperty('maptiksTrackcode');
+          var maptiksTrackcode = maptiksSet ? app.data.getWebAppData().getMaptiks().maptiksTrackcode : 'c311cf16-ad79-42b1-97f9-f433be6c8b00'; // replace with dev trackcode
+          var maptiksId = maptiksSet ? app.data.getWebAppData().getMaptiks().maptiksId : "demo";
```

Comment:
I don't think I can send that out from demo.html. I could work that into the "story-loaded-map" topic (or a custom topic), if that's preferred. Is the idea to have a single generic Maptiks.js that will be accessed by all templates?